Drag.py

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env python
+import rospy
+from geometry_msgs.msg import WrenchStamped
+from nav_msgs.msg import Odometry
+# Initialized ros parameters and global values (feel free to tweak, values were empirically gained through trial/error)
+rospy.set_param('LINEAR_FORCE', 10)
+rospy.set_param("TORQUE", 5)
+rospy.set_param('ForceDown', -20)
+rospy.set_param('TimeOfForceDown', 10)
+Linear_Drag_X = 0
```

Comment:
Consider naming things in a consistent way

Variables: `snake_case`
Functions: `snake_case`
Classes: `CamelCase`